docs/examples/watermark/image.vue

```diff
@@ -0,0 +1,5 @@
+<template>
+  <el-watermark :width="50" :height="50" image="/apple-touch-icon.png">
```

Comment:
it is better to use a SVG image like "https://element-plus.org/images/element-plus-logo-small.svg" , to make it compatible in dark mode

Reply:
Thanks for the advice,I finished it.
<img width="794" alt="image" src="https://github.com/element-plus/element-plus/assets/22388686/6325886d-5146-4328-acc7-976c90883f43">